uber/models.py

```diff
@@ -897,6 +905,9 @@ def min_badges_addable(self):
 
 
 class Attendee(MagModel, TakesPaymentMixin):
+    watchlist_id = Column(UUID, ForeignKey('watch_list.id', ondelete='set null'), unique=True, nullable=True, default=None)
+    watchlist_entry = relationship('WatchList', backref=backref('attendee', load_on_pending=True), single_parent=True, foreign_keys=watchlist_id, uselist=False)
```

Comment:
FYI, you don't need to set ``single_parent=True`` or ``foreign_keys=watchlist_id`` here.  I assume you set ``foreign_keys=watchlist_id`` is because we say ``foreign_keys=group_id`` in the ``group`` definition below.  However, the reason why that's necessary there is because there are two different foreign keys tying together that ``Attendee`` and ``Group`` tables and therefore SQLAlchemy cannot guess which one we want to use for the ``relationship``.  In this case there's only one foreign key linking ``Attendee`` and ``WatchList``, so it's okay to not specify which one you mean.

With all of that being said, we typically define our ``relationship`` on the parent instead of the child (except in cases where there are two different relationships such as with attendee/group).  So we should probably define this relationship on the ``WatchList`` class instead of here.  In theory it makes no difference but in practice almost all of the examples I find online in both the SQLAlchemy documentation and in StackOverflow Q&As define relationships on the parent, so for the sake of debugging any problems which occur I tend to want to make our code look as much like what we see on the internet as possible to minimize the differences when searching for errors and code samples and such.